### DISCUSSION TITLE
Live TIming Toggle columns on-off

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#246] [Feature requests] [answered]
Live Timing is amazing and now we have the OBC speedometer, this detail can become duplicated. Therefore ability to toggle columns on Live Timing window between on-off would be great. This will effect userse and their screen layouts too and can expand or shrink the width as they see fit.

Note that I included letters and numbers, as sometimes it is hard to remember letters and being that numbers are in sequence, it can be easier to 'play' to find the one you want.

![image](https://user-images.githubusercontent.com/22590445/194541185-b4f4511f-099a-4e93-b1e2-ed6e4cd2eb64.png)

**Comment by mhpe-dev** [comment]
I would also love more customizability in the live timing window. In my screen layout, I have space for a vertical bar with just three columns: driver names, minisectors, and tires. So a simple checkbox/toggle for each column would already be very helpful. Additionally, options to re-order columns could be implemented.

**Comment by mozjag** [comment]
I'd split the header group up into 3 separate toggles: [session info] [climate stats] [buttons]. I could live without separate shortcut keys as long as there's a toggle under (a submenu in) the settings (cogwheel) button.

On a side note, the only button that's not currently duplicated under the settings button is the one for the FIA documents, though I'll probably have that open in a browser window anyway ;-)

**Comment by orrc** [comment, marked as answer]
This should be mostly possible now in [v1.15.0](https://multiviewer.app/changelog).
